src/main/kotlin/no/nav/helse/spenn/rest/api/v1/AllowUserCheckFilter.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.helse.spenn.rest.api.v1
+
+import no.nav.security.oidc.context.OIDCRequestContextHolder
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import org.springframework.web.filter.OncePerRequestFilter
+import javax.servlet.FilterChain
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+
+@Component
+class AllowUserCheckFilter(val oidcRequestContextHolder: OIDCRequestContextHolder,
+                           @Value("\${api.access.requiredgroup:group1}") val requiredGroupMembership: String): OncePerRequestFilter() {
```

Comment:
Lurer på... burde vi hatt forskjellige gruppe-krav for read og update... i.e: speil-brukerne er jo kanskje ikke de samme som skal trigge en råkjøring?